go/vt/mysqlctl/cmd.go

```diff
@@ -27,9 +27,9 @@ import (
 )
 
 // CreateMysqld returns a Mysqld object to use for working with a MySQL
-// installation that hasn't been set up yet. This will generate a new my.cnf
-// from scratch and use that to call NewMysqld().
-func CreateMysqld(tabletUID uint32, mysqlSocket string, mysqlPort int32) (*Mysqld, error) {
+// installation that hasn't been set up yet. This will additionally generate
```

Comment:
"returns a Mysqld object and and a Mycnf object. The MyCnf object is ...."

Reply:
Renamed these functions to ..MysqldAndMycnf and updated the comments.